core/iwasm/include/wasm_c_api.h

```diff
@@ -417,6 +417,7 @@ struct wasm_ref_t;
 
 typedef struct wasm_val_t {
   wasm_valkind_t kind;
+  uint8_t __paddings[7];
```

Comment:
i guess these manual padding warrants comments.
for platforms with a loose alignment for 64 bit types?

Reply:
yes, the layout of 32-bit may be different from the layout of 64-bit, while in AOT compiler, it uses fixed layout and hopes they are the same.